mime_types.cc

```diff
@@ -0,0 +1,42 @@
+//
+// mime_types.cpp
```

Comment:
Per CS 130 expectations, can we add a comment linking to where this file came from (the Boost docs)?